sql/core/src/test/resources/sql-tests/inputs/named-function-arguments.sql

```diff
@@ -1,5 +1,50 @@
+-- Test for named arguments for Mask
```

Comment:
Could you add more tests on passing in all positional args and filling in the defaults?

Reply:
I think the current mask-expression.sql test already handles this.